packages/app-project/stores/Project.js

```diff
@@ -27,8 +28,8 @@ const Project = types
     owners: types.frozen([]),
     retired_subjects_count: types.optional(types.number, 0),
     slug: types.optional(types.string, ''),
-    urls: types.frozen([]),
     subjects_count: types.optional(types.number, 0),
+    urls: types.frozen([]),
```

Comment:
Looks like this was also missing test even though it was already existing, so let's add a test for it too.